### ISSUE TITLE
Testing needs to be done on Backpropagation class with regard to patterns

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sudostock/innsystems#27]
```
A simple program needs to be written such that it will take the
backpropagation class and the format class and use them to test the Apple
data set.

It needs to be look to see if the number of patterns increases or decreases
the accuracy of the net over time. Ex right now 10 patterns encompass about
1 hour of data. Will this still be able to predict two hours of data from
now? or does the predictions become horribly inaccurate after a few
minutes. Does increasing the number of patterns improve the accuracy over
time? These are just a few questions. 

Basically just run the tests, keep track of the data you were using err
what things you were testing and create a chart showing the accuracies of
the net with different number of patterns as compared to what the actual
value of the stock is.

```

Original issue reported on code.google.com by `darkbird44@gmail.com` on 11 Apr 2007 at 7:28
